docs/guides/js-waku/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: JavaScript Waku SDK
+---
+
+The [JavaScript Waku SDK](https://github.com/waku-org/js-waku) (`js-waku`) provides a TypeScript implementation of the [Waku protocol](/) designed for web browser environments. Developers can seamlessly integrate Waku functionalities into web applications, enabling efficient communication and collaboration among users using the `js-waku` SDK.
+
+## Installation
+
+Install the `js-waku` SDK using your preferred package manager:
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
```

Comment:
Why is it here? It's not essential part of `js-waku` installation.

Reply:
<img width="845" alt="Screenshot 2023-07-10 at 10 25 17" src="https://github.com/waku-org/docs.waku.org/assets/47832826/d9951460-4d85-41ba-996e-4939df074d74">

This is how the page is previewed, what are you referring to?

Reply:
I am reviewing in the Github viewer and forgot we are using docusaurus.